src/main/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/RefreshOrchestrator.java

```diff
@@ -40,26 +42,26 @@
 public class RefreshOrchestrator {
 
     private final RetrieveDataService retrieveDataService;
-
     private final RequestMappingService<UserAccessProfile> requestMappingService;
     private final ParseRequestService parseRequestService;
     private final CRDService crdService;
     private final PersistenceService persistenceService;
-
-
-    String pageSize;
-
+    private final SecurityUtils securityUtils;
 
     private String sortDirection;
+    private String sortColumn;
 
+    String pageSize;
 
-    private String sortColumn;
+    @Value("${refresh.authorisedServices}")
```

Comment:
We now have two different methods of setting the application settings in this class.  can we pick one and do the same for all: i.e. see refresh.Job.pageSize/sortDirection/sortColumn. in constructor.

Go with what ever is easier to test or least bother to adjust in current code. (ie.. your method is cleaner to read but changing others may be harder to do depending on number of existing tests).